src/output.rs

```diff
@@ -238,7 +274,13 @@ impl<'a, I: Iterator<Item = Event<'a>>> Iterator for EventFilter<'a, I> {
 					tag => tag
 				}),
 
-				Event::End(tag) => Event::End(match tag {
+				// skip text event when inside a github alter since we already
+				// emitted them when the link tag started
+				Event::Text(_) if self.inside_github_alert => {
+					continue;
+				},
```

Comment:
I'm not sure I understand what this does. Which text is hidden here? The text of the link body? The text of the blockquote? Please clarify the comment above

Reply:
I've extended to comment to read

> When we are inside a GitHub alert, we get three events: Start(Tag::Link), Text(<"link" text>), End(Tag::Link). Since we already emitted all the events we need to correctly render the alert from the Start event, we are skipping the Text and End events as long as we are inside the alert.

Hopefully, that's a bit clearer